src/uu/factor/src/numeric/traits.rs

```diff
@@ -6,9 +6,12 @@
 // * For the full copyright and license information, please view the LICENSE file
 // * that was distributed with this source code.
 
+pub(crate) use num_traits::{
+    identities::{One, Zero},
```

Comment:
clippy is complaining:
```
WARNING: `cargo clippy`: unused imports: `One`, `Zero`
```

Reply:
>_>'

Reply:
This was again the thing with `traits` being pulled into `build.rs`.  Fixed now.